cocos/animation/marionette/pose-graph/pose-nodes/apply-transform.ts

```diff
@@ -257,7 +263,7 @@ const {
 })();
 
 if (EDITOR) {
-    PoseNodeApplyTransform.prototype.getTitle = function getTitle (this: PoseNodeApplyTransform) {
+    PoseNodeApplyTransform.prototype.getTitle = function getTitle (this: PoseNodeApplyTransform): string | [string, Record<string, string>] | undefined {
```

Comment:
It seems returns wrong type.

Reply:
no, if you checkout the origininal implementation of `getTitle` method, it does be declared as `string | [string, Record<string, string>] | undefined`, though it's wierd to do like this.

https://github.com/cocos/cocos-engine/blob/bdcde2acf7917ea7e038ec3ea5f120494d0ce5a7/cocos/animation/marionette/pose-graph/foundation/pose-graph-node.ts#L30